### ISSUE TITLE
Mod conflict with Dawn API

### ISSUE THREAD

**Issue by RDKRACZ** [apace100/origins-fabric#45]
MC crashes (but the game is still logging) when loading Origins and Dawn API mod.

Reported to DawnTeamMC too - https://github.com/DawnTeamMC/DawnAPI/issues/2

crash report - https://hatebin.com/qnmeozqeaw
latest.log - https://hatebin.com/flwzviqyah - warning very long file

**Comment by Hugman76** [comment]
@apace100 how could we work this out? If you want, I could add some kind of method to my API to allow some kind of compatibility.

Affected mixins:
- [in Origins](https://github.com/apace100/origins-fabric/blob/8f1efeb215c19d782cb5927e3ef33dd8edc6dd0e/src/main/java/io/github/apace100/origins/mixin/BackgroundRendererMixin.java#L30)
- [in the Dawn API](https://github.com/DawnTeamMC/DawnAPI/blob/80f5ddcddebef3c08878f704a0d1902e3ee1b533/src/main/java/com/hugman/dawn/mod/mixin/BackgroundRendererMixin.java#L17)

**Comment by apace100** [comment]
This is interesting. I thought it would be difficult at first and we'd need some additional library mod, but it seems the incompatibility really isn't necessary - you are targetting the blindness check, while I'm targetting the night vision check. I've added an `ordinal = 1` to my Mixin target to only target the night vision check. I sent you a PR which changes your mixin to only target the Blindness check: I've tested the two changed builds in a clean environment, and the mods are compatible with these changes. The functionality of both mods should not be affected.
